roxie/ccd/ccdserver.cpp

```diff
@@ -390,13 +390,13 @@ static const StatisticsMapping keyedJoinStatistics({ StNumServerCacheHits, StNum
                                                     StNumIndexSkips, StNumIndexNullSkips, StNumIndexMerges, StNumIndexMergeCompares,
                                                     StNumPreFiltered, StNumPostFiltered, StNumIndexAccepted, StNumIndexRejected,
                                                     StNumIndexRowsRead, StNumDiskRowsRead, StNumDiskSeeks, StNumDiskAccepted,
-                                                    StNumBlobCacheHits, StNumLeafCacheHits, StNumNodeCacheHits,
+                                                    StNumBlobCacheHits, StNumLeafCacheHits, StNumNodeCacheHits, StNumPreloadCacheHits,
```

Comment:
Can preload hits happen per activity?

Reply:
I think so, I added because I hit an error.

Reply:
Should StNumPreloadCacheHits be removed now

Reply:
yes